source/libnormaliz/cone.cpp

```diff
@@ -1550,6 +1556,8 @@ void Cone<Integer>::prepare_input_generators(map< InputType, vector< vector<Inte
                 LatticeGenerators.append(it->second);
                 if(BasisMaxSubspace.nr_of_rows()>0)
                     LatticeGenerators.append(BasisMaxSubspace);
+                Generators.append(it->second);
+                break;
```

Comment:
This preserves semantics by making explicit what happens; but please verify that this is actually the intended behavior.